nefertari/authentication/models.py

```diff
@@ -131,7 +131,7 @@ def get_authuser_by_userid(cls, request):
         Used by Ticket-based auth. Is added as request method to populate
         `request.user`.
         """
-        userid = authenticated_userid(request)
+        userid = request.authenticated_userid
```

Comment:
tests/test_authentication/test_models.py::TestModelHelpers::()::test_lower_strip
  /Users/nikitagromov/workspace/cerri/cerri-api/nefertari/nefertari/authentication/models.py:5: DeprecationWarning: authenticated_userid: As of Pyramid 1.5 the "pyramid.security.authenticated_userid" API is now deprecated.  It will be removed in Pyramid 1.8.  Use the "authenticated_userid" attribute of the Pyramid request instead.
    from pyramid.security import authenticated_userid, forget